hooks/playbooks/ironic_network.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Create network for ironic provisioning
+  hosts: "{{ cifmw_target_hook_host | default('localhost') }}"
+  gather_facts: false
+  vars:
+    _namespace: openstack
+    _subnet_range: '172.20.1.0/24'
+    _subnet_gateway: '172.20.1.1'
```

Comment:
just another question: are those values always valid? I mean, are they independent from any underlying networking in OCP?

Reply:
Won't I be able to override those with `extra_vars`?

```
pre_tests:
  - name: 02 Create ironic provisioning network
    type: playbook
    source: ironic_network.yml
    extra_vars:
      _subnet_range: 172.20.254.0/24'
      _subnet_gateway: '172.20.244.1'
```

Reply:
It seems so, but the question was more among the line "couldn't they be dynamically identified by some other configuration", but it seems it's not the case